retrofit2/src/main/java/com/linecorp/armeria/client/http/retrofit2/ArmeriaRetrofitBuilder.java

```diff
@@ -0,0 +1,221 @@
+    private Function<String, HttpClient> newClientFunction = DEFAULT_NEW_CLIENT_FUNCTION;
+
+    /**
+     * Creates a {@link Retrofit.Builder} with {@link ArmeriaCallFactory} using the specified
+     * url.
+     *
+     * @see Builder#baseUrl(String)
+     */
+    public ArmeriaRetrofitBuilder(String baseUrl) {
+        String path = URI.create(baseUrl).getPath();
+        if (!path.isEmpty() && !SLASH.equals(path.substring(path.length() - 1))) {
+            throw new IllegalArgumentException("baseUrl must end in /: " + baseUrl);
+        }
+        this.baseUrl = baseUrl;
+        retrofitBuilder = new Retrofit.Builder();
+    }
+
+    /**
+     * Creates a {@link Retrofit.Builder} with {@link ArmeriaCallFactory} using the specified
+     * {@link HttpClient} instance.
+     */
+    public ArmeriaRetrofitBuilder(HttpClient baseHttpClient) {
+        this(baseHttpClient, SLASH);
+    }
+
+    /**
+     * Creates a {@link Retrofit.Builder} with {@link ArmeriaCallFactory} using the specified
+     * {@link HttpClient} instance.
+     */
+    public ArmeriaRetrofitBuilder(HttpClient baseHttpClient, String basePath) {
+        final String path = baseHttpClient.uri().getPath();
+        if (!path.isEmpty() && !SLASH.equals(path)) {
+            throw new IllegalArgumentException(
+                    "ArmeriaRetrofitBuilder doesn't support http client's uri contains any path element," +
+                    " current path: " + path +
+                    ". If you want to using uri with path, please using constructor with basePath argument.");
+        }
+        if (!basePath.isEmpty()) {
+            if (!SLASH.equals(basePath.substring(0, 1))) {
+                throw new IllegalArgumentException("basePath must begin with /: " + basePath);
+            }
+            if (!SLASH.equals(basePath.substring(basePath.length() - 1))) {
+                throw new IllegalArgumentException("basePath must end in /: " + basePath);
+            }
+        }
+        this.baseHttpClient = baseHttpClient;
+        this.basePath = basePath;
+        retrofitBuilder = new Retrofit.Builder();
+    }
+
+    /**
+     * Set different group name prefix for avoiding conflict of endpoint which starts with __group__.
+     */
+    public ArmeriaRetrofitBuilder groupPrefix(String groupPrefix) {
+        if (!GROUP_PREFIX_PATTERN.matcher(groupPrefix).matches()) {
+            throw new IllegalArgumentException(
+                    "groupPrefix: " + groupPrefix + " (expected: " + GROUP_PREFIX_PATTERN.pattern() + ')');
+        }
+        this.groupPrefix = groupPrefix;
+        return this;
+    }
+
+    /**
+     * Set a function for customizing HttpClient.
+     */
+    public ArmeriaRetrofitBuilder newClientFunction(Function<String, HttpClient> newClientFunction) {
+        this.newClientFunction = newClientFunction;
+        return this;
+    }
+
+    /**
+     * @see Retrofit.Builder#addCallAdapterFactory(Factory)
```

Comment:
Having only `@see` tag makes the API documentation look bad. Could you at least add one sentence? (Same applies to the other methods.)